src/Aquifer.JsEngine/Aquifer.JsEngine.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net8.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="JavaScriptEngineSwitcher.V8" Version="3.24.2" />
+        <PackageReference Include="Microsoft.ClearScript.Complete" Version="7.4.5" />
+    </ItemGroup>
+
+    <ItemGroup>
+        <EmbeddedResource Include="..\Aquifer.Tiptap\dist\TiptapUtilities.js" Link="Tiptap\TiptapUtilities.js" />
+    </ItemGroup>
+
+    <ItemGroup>
+        <ProjectReference Include="..\Aquifer.Tiptap\Aquifer.Tiptap.esproj" />
```

Comment:
This project reference required me to turn off a build warning but the reference is necessary in order to ensure that `TiptapUtilities.js` is built and available before this project builds.